ui/apps/platform/src/Containers/Policies/Wizard/Step4/PolicyScopeCard.tsx

```diff
@@ -121,6 +121,8 @@ function PolicyScopeCard({
                                     isOpen={isClusterSelectOpen}
                                     selections={value.cluster || scope?.cluster}
                                     placeholderText="Select a cluster"
+                                    hasInlineFilter
```

Comment:
The implementation on the Dashboard that was referenced in the Jira ticket is simply a regular deprecated, PatternFly `<Select>` element with the `hasInlineFilter` property. The former uses a checkbox select, which is not relevant here since the UX calls for single-selection of deployments.